tool/tsh/common/autoupdate.go

```diff
@@ -27,21 +27,16 @@ import (
 
 type autoUpdateCommand struct {
 	update *managedUpdatesUpdateCommand
-	clean  *managedUpdatesCleanCommand
 }
 
-func newAutoUpdateCommand(app *kingpin.Application) *autoUpdateCommand {
+func newUpdateCommand(app *kingpin.Application) *autoUpdateCommand {
 	root := &autoUpdateCommand{
 		update: &managedUpdatesUpdateCommand{},
-		clean:  &managedUpdatesCleanCommand{},
 	}
 
-	parentCmd := app.Command("autoupdate", "Client-tools managed updates commands")
-
-	root.clean.CmdClause = parentCmd.Command("clean",
-		"Removes locally installed client tools updates from the Teleport home directory.")
-	root.update.CmdClause = parentCmd.Command("update",
-		"Update client tools to the latest version defined by the cluster configuration")
+	root.update.CmdClause = app.Command("update",
+		"Update client tools to the latest version defined by the cluster configuration.")
```

Comment:
nit: users might not know what client tools are

```suggestion
		"Update client tools (tsh, tctl) to the latest version defined by the cluster configuration.")
```